packages/search/src/ago/encode-ago-query.ts

```diff
@@ -0,0 +1,57 @@
+import { getSortField } from "./helpers/sort";
+import { createFilters, handleFilter } from "./helpers/filters";
+import { createAggs } from "./helpers/aggs";
+
+/**
+ * Construct a query object with filters, and queryParams sent by hub indexer
+ * @param queryObject any
+ */
+export function encodeAgoQuery(queryParams: any = {}) {
```

Comment:
all `export`ed functions show up in the documentation unless they are added to the explicit blacklist below.

i think it'd be appropriate to hide a lot of your shared utility functions, but i'll leave it up to you to decide exactly which.

https://github.com/Esri/hub.js/blob/da48fbc9e13ce46223086cdb9082a39fd4128d18/docs/build-typedoc.js#L138

```ts
const blacklist = [ "encodeAgoQuery", "etc" ];
```